app/views/councillor_contributions/new.html.haml

```diff
@@ -5,5 +5,9 @@
       - @councillor_contribution.suggested_councillors.each do |s|
         = render 'contribution_form_input', f: f, suggested_councillor: s
       %button{formaction: new_authority_councillor_contribution_path(@authority.short_name_encoded), class: "button"} Add another councillor
+    .councillor-contribtution-source
+      %p Where did you find this information? please include a link if possible
```

Comment:
"please include a link if possible" should be "Please include a link if possible".

But maybe something a little more conversational? "If you found this information online, please provide the URL where we can check it." or something that hints at why we want the data?